packages/RatPack.package/RPTests.class/instance/testPutWithIncompleJson.st

```diff
@@ -0,0 +1,16 @@
+test - REST operations
+testPutWithIncompleJson
+	
+	| url urlId putResult postResult testJson incompleteTestJson resultTestJson |
+	app registerModel: RPTestModel.
+	url := app class apiRoutePrefix, (RPTestModel name).
+	testJson := RPTestHelper getExampleJson.
```

Comment:
Could you inline this (-> just make line 11 say `... app content: RPTestHelper getExampleJson`)? Reduces lines and method variables.

Actually, I don't like that the JSON strings are in class methods in `RPTestHelper`. If they are capsuled in a class, it should be in `RPTests`. And the JSON these functions return are often shorter than the method call itself. I see that it's somewhat reducing code duplication, but since every test should be independent anyways, I don't think that's really necessary. @jakobmache @Nina510 @essal0815 what do you think? I'd refactor this in another PR.

Reply:
Well taking it to the class side of RPTests  I agree. But having a method, helps me, because I don't always want to check what names or datatypes where specified in RPTestModel...
So I know I have a Json, I can rely on.

Reply:
Currently, you have to remember that
- there is a method that gives you JSON describing an instance of RPTestModel
- it's a class method of RPTestHelper, although you want json that describes an RPTestModel
- it's called `getExampleJson`, although accessors are not usually prefixed with "get" in smalltalk

What you would have to remember otherwise:
- RPTestModel has a field "salary", which is intuitively an integer
- RPTestModel has a field "name", which is intuitively a string

I don't really agree that it's easier. But if you want to keep it, we can keep it.

I still think that it's misplaced in `RPTestHelper` though. Currently, someone new reading the code literally has the information that it will return some kind of json. He does not know what the json contains, whether its complete or incomplete or even what model it's meant to describe.

If we want to keep the methods, I'd like `RPTestModel>>fullJson` and `RPTestModel>>partialJson` much more. Then again, you are defining what you want as result for the request in this method.

Reply:
I would want to have the methods but on the RPTestModel. But were they were it made sense for me since there is all the convenience stuff needed for the tests.
In my opinion small talk makes you always dive deep into the methods to understand what you get xD